README.md

```diff
@@ -14,6 +14,10 @@ All modules in this repository adhere to the module strutucture defined in the t
 - [Module Development](https://seed-farmer.readthedocs.io/en/latest/module_development.html)
 - [Module Manifest Guide](https://seed-farmer.readthedocs.io/en/latest/manifests.html)
 
+#### Industry-Specific SeedFarmer Module Repositories
+- [ADDF](https://github.com/awslabs/autonomous-driving-data-framework): A collection of modules for Scene Detection, Simulation (mock), Visualization, Compute, Storage, Centralized logging, etc.
```

Comment:
The underlying modules such as Compute, storage etc will be available in IDF.

Reply:
Isn't this clear in the README already: https://github.com/awslabs/idf-modules?tab=readme-ov-file#general-information